codebuild/bin/install_default_dependencies.sh

```diff
@@ -65,9 +65,10 @@ if [[ "$S2N_LIBCRYPTO" == "boringssl" && ! -d "$BORINGSSL_INSTALL_DIR" ]]; then
 fi
 
 # Download and Install AWS-LC
-if [[ "$S2N_LIBCRYPTO" == "awslc" && ! -d "$AWSLC_INSTALL_DIR" ]]; then
+if [[ ("$S2N_LIBCRYPTO" == "awslc") || ( "$TESTS" == "integrationv2" || "$TESTS" == "ALL" ) && ! -d "$AWSLC_INSTALL_DIR" ]]; then
```

Comment:
Is it necessary to add the test conditions? Since we already have some integration tests that run on AWS-LC, I think the previous condition should be sufficient to get coverage?

Reply:
> Is it necessary to add the test conditions?
Not anymore now that s2n's internal Kyber implementation has been removed in https://github.com/aws/s2n-tls/pull/4283 

Fixed